src/almalinux/8/helix/amd64/Dockerfile

```diff
@@ -21,9 +21,6 @@ RUN dnf upgrade --refresh -y \
         file \
     && dnf clean all
 
-RUN python3 -m pip download --no-deps helix-scripts --index-url https://dnceng.pkgs.visualstudio.com/public/_packaging/helix-client-prod/pypi/simple \
```

Comment:
Are there any prereqs that aren't needed since the script is no longer run?  For example, there is this comment - https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1336/files#diff-f8d93a264641a0bbae6526f0d72e990bfb0bd1e61b581daa07cf4cd5a84a7371R11

Reply:
Python still needs to be installed. I've updated the comment to be more appropriate for why we're installing that specific package.

Reply:
/fyi this requirement for Helix containers won't go away until and unless we retire the Helix Python code in dotnet/arcade. the Helix SDK regularly adds post-commands relying on Python